libtelemetry/src/main/java/com/mapbox/android/telemetry/ChinaCertificatePins.java

```diff
@@ -26,6 +26,57 @@
           add("sha256/+O+QJCmvoB/FkTd0/5FvmMSvFbMqjYU+Txrw1lyGkUQ=");
           add("sha256/zUIraRNo+4JoAYA7ROeWjARtIoN4rIEbCpfCRQT6N6A=");
           add("sha256/r/mIkG3eEpVdm+u/ko/cwxzOMo1bk4TyHIlByibiA5E=");
+
+          add("sha256/T4XyKSRwZ5icOqGmJUXiDYGa+SaXKTGQXZwhqpwNTEo=");
```

Comment:
Are there any security implications to hardcoding certificate pins?

Reply:
Great question. I am not too sure. This system does include a blacklist to turn off any compromised hashes and we have always had the pins hardcoded (see above this point in the code), but something we should discuss with the security team and talk about any solutions if there are problems. I'll ping them.